lib-opensource/emqx_management/test/emqx_mgmt_SUITE.erl

```diff
@@ -294,6 +294,14 @@ t_listeners_cmd_new(_) ->
        "Stop mqtt:wss:external listener on 0.0.0.0:8084 successfully.\n",
        emqx_mgmt_cli:listeners(["stop", "mqtt:wss:external"])
       ),
+    ?assertEqual(
+       emqx_mgmt_cli:listeners(["restart", "mqtt:tcp:external"]),
+       "Restarted mqtt:tcp:external listener successfully.\n"
+      ),
+    ?assertEqual(
+       emqx_mgmt_cli:listeners(["restart", "mqtt:ssl:external"]),
+       "Restarted mqtt:ssl:external listener successfully.\n"
```

Comment:
for test coverage, could you add another assertion to match the `Failed to restart` printout ?
e.g. by giving an unknown listener.